seshdash/utils/model_tools.py

```diff
@@ -130,3 +133,24 @@ def get_status_card_items(site):
     except Exception  ,e :
         looger.error(e)
         pass
+
+def get_site_measurements(site):
+
+    site_measurements = site.site_measurements
+    # getting all measurement fields
+    site_measurements_fields = site_measurements._meta.get_fields()
+    site_measurements_items = []
+    for field in site_measurements_fields:
+        site_measurements_items.append(getattr(site_measurements, field.name))
+
+    #removing non char items
+    for i,item in enumerate(site_measurements_items):
```

Comment:
ok but in line 103. You alreadu have a nice verbose name in the list. Why aren't you using that?